### PR TITLE
JBPM-8598 - Global Variables property is missing translation key

### PR DESCRIPTION
**Pull request by Josephblt**
https://issues.jboss.org/browse/JBPM-8598

Quick fix for missing translation.

### CODE REVIEW

**Comment by Josephblt** [comment]
Jenkins execute full downstream build

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Comment by Josephblt** [comment]
Jenkins execute full downstream build

**Comment by Josephblt** [comment]
Jenkins execute full downstream build

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants.properties [hunk 1]
```diff
@@ -193,6 +193,8 @@ org.kie.workbench.common.stunner.bpmn.definition.property.diagram.ProcessInstanc
 org.kie.workbench.common.stunner.bpmn.definition.property.diagram.ProcessInstanceDescription.description=This attribute is used to annotate Process Instances
 org.kie.workbench.common.stunner.bpmn.definition.property.diagram.Version.label=Version
 org.kie.workbench.common.stunner.bpmn.definition.property.diagram.Version.description=The process version
+org.kie.workbench.common.stunner.bpmn.definition.property.diagram.GlobalVariables.label=Global Variables
+org.kie.workbench.common.stunner.bpmn.definition.property.diagram.GlobalVariables.description=This property is used to declare variables with global scope.
```
Comment (romartin) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants.properties]: minor thing, but I would just remove the last dot symbol... as this meesage usually appears where no other sentence, just as  a title, no title to end up with a dot IMO, WDYT?

Comment (Josephblt) [reply kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants.properties]: All good now. Removed the dot.

**Comment by Josephblt** [comment]
Jenkins execute full downstream build

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants.properties (hunk 1 above)
Comment (LuboTerifaj) [reply kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants.properties]: It's the description, not the title. Therefore I would let the dot there. I would also choose the same approach as the other descriptions use in this file, which is the sentence containing dot.

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants.properties [hunk 2]
```diff
@@ -193,6 +193,8 @@ org.kie.workbench.common.stunner.bpmn.definition.property.diagram.ProcessInstanc
 org.kie.workbench.common.stunner.bpmn.definition.property.diagram.ProcessInstanceDescription.description=This attribute is used to annotate Process Instances
 org.kie.workbench.common.stunner.bpmn.definition.property.diagram.Version.label=Version
 org.kie.workbench.common.stunner.bpmn.definition.property.diagram.Version.description=The process version
+org.kie.workbench.common.stunner.bpmn.definition.property.diagram.GlobalVariables.label=Global Variables
+org.kie.workbench.common.stunner.bpmn.definition.property.diagram.GlobalVariables.description=This property is used to declare variables with global scope
```
Comment (LuboTerifaj) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants.properties]: The sentence with dot or without dot should be unified across the whole file. See also comment above.

Comment (LuboTerifaj) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants.properties]: The sentence with dot or without dot should be unified across the whole file. See also comment above.

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants.properties (hunk 1 above)
Comment (Josephblt) [reply kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants.properties]: Yeah I agree. So the dot is back, hehe. :)

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants.properties (hunk 2 above)
Comment (Josephblt) [reply kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants.properties]: Yeah I agree. But it is somewhat confusing. Some of the descriptions have the dot some don't. It's hard. But I don't think this should stop us from merging this PR. I suggest we solve this in another JIRA.

**Comment by Josephblt** [comment]
Jenkins execute full downstream build

**Comment by Josephblt** [comment]
Jenkins execute full downstream build

**Comment by Josephblt** [comment]
Jenkins execute full downstream build

**Comment by Josephblt** [comment]
Jenkins execute full downstream build

**Comment by Josephblt** [comment]
Hello @romartin. I think this is good. Build failed but this is OK.

**Comment by romartin** [comment]
Jenkins execute full downstream build

**Comment by romartin** [comment]
please @LuboTerifaj missing your final review here. Thx!

**Comment by Josephblt** [comment]
Jenkins execute full downstream build

**Comment by romartin** [comment]
Artifacts properly generated, just missing @LuboTerifaj 's final review here, thx guys

**Comment by Josephblt** [comment]
Jenkins execute full downstream build

**Comment by Josephblt** [comment]
Hello @LuboTerifaj. Just missing your review here. Thanks.

**Pull request merged**